glusterd2/commands/peers/deletepeer.go

```diff
@@ -61,13 +62,6 @@ func deletePeerHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
```

Comment:
Please reintroduce the `isAlive` soft check, in addition to moving `DeletePeer` below. That double check helps

Reply:
sure @prashanthpai